<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="editShotTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>631</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editShotSavePushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Tallenna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editShotCancelPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peruuta</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaadot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editShotPopulatePushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Täytä</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editShotByComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editShotAnimalComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editShotAgeComboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="editShotGenderComboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="editShotDateEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editShotLocationLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editShotWeightLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="editShotAdditionalInfoPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>330</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>50</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaataja</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>80</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Kaatopäivä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Paikka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Eläin</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>90</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ikäryhmä</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>100</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sukupuoli</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>60</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Paino (kg)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>80</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Lisätietoja</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotByLabel_18">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Käyttö</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editShotUsage2ComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="editShotUsage2CheckBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Toinen Käyttö</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editShotUsageComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="editShotUsage2PortionSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="shotByLabel_19">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>420</y>
     <width>80</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Käytön osuus</string>
   </property>
  </widget>
  <widget class="QLabel" name="shotByLabel_20">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Toisen käytön osuus</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="editShotUsagePortionSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>440</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>25</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
